packages/client3/constants/metadata/mainnet.ts

```diff
@@ -12,6 +12,7 @@ import type { PoolMetadata } from "./types";
 
 export const mainnetMetadata: Record<string, PoolMetadata> = {
   "0xd43a4f3041069c6178b99d55295b00d0db955bb5": {
+    artist: "somethinbg",
```

Comment:
Lol whatever, but `something`
```suggestion
    artist: "something",
```